app/models/participant_profile/ecf.rb

```diff
@@ -79,6 +79,15 @@ def relevant_induction_record(lead_provider:)
         .first
     end
 
+    def role
```

Comment:
fyi if you implemented `role` on the subclasses there would be no need for a case statement

Reply:
I moved it in the ParticipantProfile class and overriding it in the subclasses.